core/src/main/java/com/android/volley/toolbox/NetworkUtility.java

```diff
@@ -50,7 +50,7 @@ private NetworkUtility() {}
     /** Logs requests that took over SLOW_REQUEST_THRESHOLD_MS to complete. */
     static void logSlowRequests(
```

Comment:
Let's rename this to "logRequestSummary" and update the Javadoc to reflect, e.g.:

"Logs a summary about the request when debug logging is enabled."